rfcs/input-component.md

```diff
@@ -125,12 +123,6 @@ export default () => {
   Defines styles for inputs that are grouped with other controls.
 * `size: 'default' | 'compact'`
   Defines the size of an input control.
-* `onChange: Function` - Optional
```

Comment:
We don't do any overrides for the `onChange` on the stateless input but wouldn't it be confusing when we don't say how the `onChange` handler supposed to be passed in? For example, is it on the top level or is it in `overrides` for the BaseInput? Currently, we do pass through all the top-level props to the BaseInput (maybe we need to change it and make it consistent for all components), but we don't pass through any undocumented props to the subcomponents in others, like `Modal` for example.

Reply:
I see - what are your thoughts on this @schnerd @blackswanny @abmai?

Reply:
customer shouldn't know, what components we re-use under the hood. I would rather have onChange passed through parent component, cause RFC represents interface only for the very exact component. So may be experienced customers will find the way around via overrides, but it's not for veryone's usage